api/database/seeders/SkillSeeder.php

```diff
@@ -72,7 +72,7 @@ function ($model) {
                         'en' => $model->description->en,
                         'fr' => $model->description->fr,
                     ],
-                    'category' => $model->category,
+                    'category' => $model->category->value,
```

Comment:
Could you update the query comment above as well?

Reply:
Yep, can do https://github.com/GCTC-NTGC/gc-digital-talent/pull/11523/commits/458dd053b8ceb4a335215dd66cfce1ead9ae04b1